README.md

```diff
@@ -344,6 +344,31 @@ These settings can be changed at any time.
 * `settings.updates.version-lock`: Controls the version that will be selected when you issue an update request.  Can be locked to a specific version like `v1.0.0`, or `latest` to take the latest available version.  Defaults to `latest`.
 * `settings.updates.ignore-waves`: Updates are rolled out in waves to reduce the impact of issues.  For testing purposes, you can set this to `true` to ignore those waves and update immediately.
 
+#### Network settings
+
+These settings will configure the networking behavior of the following services:
+* For all variants:
+    * [containerd.service](packages/containerd/containerd.service)
+    * [host-containerd.service](packages/host-ctr/host-containerd.service)
+* For Kubernetes variants:
+    * [kubelet.service](packages/kubernetes-1.18/kubelet.service)
+* For ECS variants:
+    * [docker.service](packages/docker-engine/docker.service)
+    * [ecs.service](packages/ecs-agent/ecs.service)
+
+* `settings.network.http-proxy`: The HTTP proxy server to be used by services listed above.
+* `settings.network.https-proxy`: The HTTPS proxy server to be used by services listed above.
+* `settings.network.no-proxy`: A list of hosts that are excluded from proxying.
```

Comment:
```suggestion
* `settings.network.no-proxy`: A comma-separated list of hosts that are excluded from proxying.
```

Reply:
Specifying "comma-separate" gives the impression that this setting is a single line string when in fact it should be specified with a toml array of strings. I think "A list" is appropriate here. 

(The description for `settings.ecs.logging-drivers` also says "list")

Now if I think about it. Is it a bad idea to make `no-proxy` a single line string setting and say it should be a comma-separated list?... I'm somewhat inclined to say we shouldn't.

Reply:
Ah, right, sorry.  List!  We should definitely represent a list as a list, not a weird string.